src/main/java/nl/tudelft/lifetiles/graph/model/GraphContainer.java

```diff
@@ -55,8 +66,12 @@ public GraphContainer(final Graph<SequenceSegment> graph) {
      */
     private void alignGraph() {
         UnifiedPositionTraverser upt = new UnifiedPositionTraverser();
-        EmptySegmentTraverser est = new EmptySegmentTraverser();
-        graph = est.addEmptySegmentsGraph(upt.unifyGraph(graph));
+        if (Settings.getBoolean(SETTING_EMPTY)) {
+            EmptySegmentTraverser est = new EmptySegmentTraverser();
+            graph = est.addEmptySegmentsGraph(upt.unifyGraph(graph));
```

Comment:
`upt.unifyGraph(graph)` is double code. Better use this structure: `new ReferencePositionTraverser(reference).referenceMapGraph(graph);`. The `graph =` isn't needed.